<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scanerView</class>
 <widget class="QDialog" name="scanerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="3">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Настройки</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Качество документа</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboQuality"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Цветность документа</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboColor"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="buttonView">
     <property name="text">
      <string>Просмотр документа</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStartScan">
        <property name="text">
         <string>Начать сканирование</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonExit">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="3" colspan="3">
    <widget class="QLabel" name="labelPreView">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="buttonSave">
     <property name="text">
      <string>Сохранить документ</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Сканирование документа</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="buttonPrintDoc">
     <property name="text">
      <string>Печать документа</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
